### PR TITLE
adding PKS solution description and link

### PR DESCRIPTION
**Pull request by heckj**
Adding Pivotal's PKS to the list of hosted solutions

### CODE REVIEW

**Comment by heckj** [comment]
@bradtopol since I made this update, would appreciate a second set of eyes reviewing the content and approving it. 

/assign @bradtopol

File: docs/setup/pick-right-solution.md [hunk 1]
```diff
@@ -61,6 +61,8 @@ a Kubernetes cluster from scratch.
 
 * [Kubermatic](https://www.loodse.com) provides managed Kubernetes clusters for various public clouds, including AWS and Digital Ocean, as well as on-premises with OpenStack integration.
 
+* [Pivotal Container Services](https://pivotal.io/platform/pivotal-container-service) provides enterprise-grade Kubernetes for both on-premises and public clouds.  PKS enables on-demand provisioning of Kubernetes clusters, multi-tenancy and fully automated day-2 operations.
```
Comment (bradtopol) [inline docs/setup/pick-right-solution.md]: Kind of funny that they use PKS as their acronym even though they are called Pivotal Container Services.  I guess when they want you to think Container service, they want you to think something that starts with a K :-)

**Comment by bradtopol** [comment]
Great job!
/lgtm

**Pull request merged**